### DISCUSSION TITLE
Support for Android

### DISCUSSION THREAD

**Discussion by ShouYuman779** [ventoy/Ventoy#892] [Ideas]
I would like to suggest to make android apk so you can make bootable usb from android phone

**Comment by chromer030** [comment]
Is it possible to partition a USB drive without root permission on android ?

**Reply by blaked333** [reply]
I can't understand why you see any benefits to this other than maybe an alternative to a PC if you don't have one?

**Comment by ufgf** [comment]
As far as I know, this is only possible with a root access, and DriveDroid (there should also be support at the kernel level)
